Dockerfile

```
@@ -0,0 +1,27 @@
+FROM python:3

```

Will bring the base image size down from 921MB to 126MB.
```suggestion
FROM python:3-slim
```